homeassistant/components/wake_word/__init__.py

```diff
@@ -149,5 +149,5 @@ def websocket_entity_info(
 
     connection.send_result(
         msg["id"],
-        {"wake_words": entity.supported_wake_words},
+        {"wake_words": (await entity.get_supported_wake_words())},
```

Comment:
We should wrap this in a timeout because we don't know how the implementation works.